contributed_definitions/NXsensor_scan.nxdl.xml

```diff
@@ -60,158 +60,158 @@
         </field>
         <group name="experiment_identifier" type="NXidentifier" recommended="true">
             <doc>
-                 The unique identifier for the entry. The identifier is mainly lab-defined and
-                 can be a combination of the sample name, date and time, experiment condition
-                 (such as temperature) or instrument-generated unique identifier.
+                The unique identifier for the entry. The identifier is mainly lab-defined and
+                can be a combination of the sample name, date and time, experiment condition
+                (such as temperature) or instrument-generated unique identifier.
             </doc>
         </group>
         <group name="collection_identifier" type="NXidentifier" optional="true">
             <doc>
-                 The unique identifier for the collection. The identifier is used to group a
-                 number of the experiments run upon the same setup and/or same sample.
+                The unique identifier for the collection. The identifier is used to group a
+                number of the experiments run upon the same setup and/or same sample.
             </doc>
         </group>
         <field name="experiment_description" recommended="true"/>
-        <field name="start_time" type="NX_NUMBER" units="NX_TIME" recommended="true"/>
```

Comment:
No, the units are not NX_TIME as this is not a time duration. Please check the definition in NXentry, and only overwrite here its optionality.